From d4b487645aad059157aae32e471cb4ee2dc9e1c5 Mon Sep 17 00:00:00 2001
From: Marvin Schmidt <marv@exherbo.org>
Date: Mon, 26 Sep 2016 08:11:21 +0200
Subject: [PATCH 2/2] tests: Remove coredump tests
Upstream: No, not a suitable solution. Improvement of tests is/was discussed at https://lists.nongnu.org/archive/html/libunwind-devel/2016-02/
msg00019.html

The coredump tests rely on the assumption that coredump files
are created in the current working directory. This assumption is
false when e.g. systemd-coredump is used.
---
 tests/Makefile.am | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index a367eed..3dcf3b1 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -56,17 +56,6 @@ if SUPPORT_CXX_EXCEPTIONS
  check_PROGRAMS_cdep += Ltest-cxx-exceptions
 endif
 
-if OS_LINUX
-if BUILD_COREDUMP
- check_SCRIPTS_cdep += run-coredump-unwind
- noinst_PROGRAMS_cdep += test-coredump-unwind
-
-if HAVE_LZMA
- check_SCRIPTS_cdep += run-coredump-unwind-mdi
-endif # HAVE_LZMA
-endif # BUILD_COREDUMP
-endif # OS_LINUX
-
 perf: perf-startup Gperf-simple Lperf-simple Lperf-trace
 	@echo "########## Basic performance of generic libunwind:"
 	@./Gperf-simple
-- 
2.10.0

